lib/vecmat.gi

```diff
@@ -1280,7 +1280,12 @@ InstallGlobalFunction(ConvertToVectorRepNC,function( arg )
                 #
                 return true;
             fi;
-            CLONE_OBJ(v,vc); # horrible hack.
+            # SWITCH_OBJ changes the mutability of v, so we make
+            # sure that if v is immutable it stays immutable.
+            if not IsMutable(v) then
+                MakeImmutable(vc);
+            fi;
+            SWITCH_OBJ(v,vc);
```

Comment:
Aha... So, does this not work in HPC-GAP? I notice that there, the code still uses `CLONE_OBJ`